hunt/app/assets/stylesheets/animals.scss

```diff
@@ -7,5 +7,6 @@
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    cursor: pointer
```

Comment:
This is okay for now because we only have one table in our application, but maybe later on we add another table and now it looks clickable when it might not be. What do you think about creating a new class like "link-row" or something, and adding that class to the table row. Then here, you can just change the cursor for `td` belonging to rows with that class?